### DISCUSSION TITLE
What is the benifit of using --quad option?

### DISCUSSION THREAD

**Discussion by imyhxy** [ultralytics/yolov5#4029] [Q&A] [answered]
Hi there, I have noticed that there is a `--quad` option in the `train.py` script which will collect 4 normal images into single big one in the collate step of the dataloader process. And I have trained *w/wo* this option and observed no significant difference on my custom dataset. Can you explain when should we use `--quad` to get a better result?

**Comment by glenn-jocher** [comment, marked as answer]
@imyhxy it's an experiment that allows for occasional usage of 2x larger images without requiring the extra GPU memory. It goes with collate_fcn4():

https://github.com/ultralytics/yolov5/blob/b7e985e397e12ea2efd19bf3b6329028fb2a4c75/utils/datasets.py#L594-L619
